fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -32,8 +32,6 @@ class ExponentiatedGradient(BaseEstimator, MetaEstimatorMixin):
     The exponentiated gradient algorithm is described in detail by
     :footcite:t:`agarwal2018reductions`.
 
-    Read more in the :ref:`User Guide <exponentiated_gradient>`.
-
```

Comment:
O_o don't we need to replace this reference with something that works?

Reply:
I couldn't find what to replace it with concept wise after looking through the user guide, so I thought it got maybe outdated. If you have an idea what to link there I'd gladly do it :)

Reply:
@adrinjalali maybe this one could be an option? https://fairlearn.org/v0.10/auto_examples/plot_mitigation_pipeline.html ?

Reply:
Yeah it sucks that we still don't have a good doc for that. The example seems not bad.

Reply:
I made a not for myself to create a better example in the future when there is time for it, and I added a link to the `plot_mitigation_pipeline` in [c79cc68](https://github.com/fairlearn/fairlearn/pull/1378/commits/c79cc68a043ec748b095b4793262d93f95f92d52).